cdapython/functions.py

```diff
@@ -2,6 +2,7 @@
 from cda_client.model.query import Query
 from os import path
 from ssl import get_default_verify_paths
+from cdapython.ConvertionMap import convertionMap
```

Comment:
Should this be ConversionMap?

Reply:
i'm doing a module import so that is the name of the file and i'm importing a dict or map into the functions file

Reply:
Oh it was miss spelled lol